src/test/resources/fixtures/single.yaml

```diff
@@ -1,8 +1,4 @@
 ---
-serial-number: 1
-hash: "someHash1"
-entry:
-  fields:
-  - "field1"
-  register: "value1"
-  text: "The Entry 1"
+entry-number: "1"
```

Comment:
I raised openregister/specification#51 for this issue. Let's just go with strings for the moment, we can change it later if necessary.